### ISSUE TITLE
Weird Playback Issues on Android

### ISSUE THREAD

**Issue by iiidefektiii** [RenderHeads/UnityPlugin-AVProVideo#558]
**Describe the bug**
I have an existing project with a ton of work in it that I am using the AVPro video with. I also have a Repository in connection with it and trying not to lose any of that. What my issue is is that playback is really weird and choppy. I have tested in a brand new project and it works fine. I have also exported all the contents of my project and imported into a new project and when I add the manifest from the origianal project it does the same thing. I have also attempted to instead of copying the manifest to the new project. Load each project as new one by one but never had the issue. I have also tried going through one setting at a time in the new project to match to original project and it seemed fine. Bit not sure if there is a setting hiding somewhere in the original cause this issue. It is very frustrating and have spent days trying to figure out why it just will not work in the original project. Maybe by viewing it you have some accross this issues in the past?

**Your Setup (please complete the following information):**
 - Unity version:  2020.1.11
 - AVPro Video version: 1.11.7
 - Operating system version: Android
 - Device model: Pixel 3 XL
 - Video specs (resolution, frame-rate, codec, file size): 
 - 1088x2224 
 - framerate: 56.96 
 - mp4
 - 41mb but also does it using the vimeo streaming

**To Reproduce**
1. I would need to send the project but it is confidential and cannot at the moment. Unless I can work something out or get consent.
2. 
3. 

**Logs**
If applicable, add error logs to help explain your problem.
No errors
**Screenshots**

**Videos**
If applicable, add a copy of your video or the URL
https://vimeo.com/494498893/0a21096974

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by AndrewRH** [comment]
Hi,
Thanks for submitting the issue.  That's very strange.
I suspect there is an issue with your video encoding.  The frame rate is very strange (56.96).  Perhaps you could try re-encoding your videos?   Do you get this problem if you use any of the sample videos we included?  You could also try enabling the OES rendering path (in Platform Specific > Android menu on the MediaPlayer).

Thanks,

**Comment by iiidefektiii** [comment]
The thing is it works perfectly fine in any other project. Which is why I am stumped. I have even tried the vimeo route and that also works streaming to avpro no issues im another app. Only the one project.

The only thing that works is export all assets as a package and import them into a new project. Then I have to import one package at a time that I am using and also setting by setting between the two and it's fine. Unless there is a setting embedded somewhere I am missing and that is why it works. If I try to make a new project and do this but copy the manifest of packages over so they all get installed it does the same thing in the new project as well.

**Comment by iiidefektiii** [comment]
@AndrewRH 

So setting android to OES rendering worked. I still find it odd that I can make this work for without change it to OES resuming in any other project but this one.

What would cause such and issue and what is OES rendering doing differently that makes it work? Like I said I also tried the vimeo api to avpro route so the video was not even in streaming assets and it still had the issue.

**Comment by AndrewRH** [comment]
Yes it sounds extremely strange how your other projects behave differently...very odd.  I have no idea why this would happen.

OES rendering is just displaying the native decoded texture, so there is less going on, so the performance is also improved.  Without OES it has to copy the OES texture to another texture (a Unity friendly texture), so there is this extra step.  I'm not sure why the stuttering would be happening though.

Are there other plugins in your scene?  Perhaps one of them is interfering somehow...What happens if you try with a blank project?

It could also be a bug in Unity - you could try upgrading to see if that fixes it.

Thanks,

**Comment by iiidefektiii** [comment]
@AndrewRH I am sending you a link to a zip of the project without the 3d data and stuff that is not allowed to go out. I have kept all packages and plugins in tact. I am sending you the link in the email I received back from support. Do you receive those?

**Comment by iiidefektiii** [comment]
Blank project works fine from the get go, unfortunatly I have been working on a project before they wanted avpro implemented for a tutorial video. I went through a whole process of exporting the entire scene as a package. Then imported into a new project. It depends on packages tho so I went through one by one and installed those packages. Still worked fine. Then opened the old broken project and new project and went through each setting one by one and tested and everything still worked. If i try the same process and just copy the package manifest over it does not work. The only issue with all of this is keeping everything intact with a repo I already have a history on.
